addon/components/sort-pane.js

```diff
@@ -80,12 +78,11 @@ export default class SortPaneComponent extends Component {
   
   @computed('sortManager.activeSortPane')
   get isActiveSortPane() {
-    return isEqual(this, get(this, 'sortManager.activeSortPane'));
-  }
-  
-  @computed('items.[]')
-  get collection() {
-    return this.args.items;
+    if (this.args.retainPlaceholderOnMoveOut) {
+      return true;
+    } else {
```

Comment:
So that backdrop-placeholder is not removed when dragging out of the column.

what happens if not there:
https://user-images.githubusercontent.com/33452443/107737638-db494c00-6d2a-11eb-9230-d029174ecd71.mp4